runtime/src/main/java/org/corfudb/runtime/collections/CorfuTable.java

```diff
@@ -263,7 +263,6 @@ public V get(@ConflictParameter Object key) {
         if (indexFunctions.isEmpty()) {
             // If there are no index functions, use the entire map
             entryStream = mainMap.entrySet().parallelStream();
-            log.debug("getByIndexAndFilter: Attempted getByIndexAndFilter without indexing");
```

Comment:
Doesn't seem related to this PR

Reply:
This seemed to be polluting the mp logs for single index tables, can submit a new PR.

Reply:
Submit a new PR, it's not related to orchestration service at all

Reply:
done.